### DISCUSSION TITLE
Adding a new site

### DISCUSSION THREAD

**Discussion by Purfview** [techmovie/easy-seed#65] [Q&A]
I started looking how to add a new site, so first not clear thing is why `seedDomSelector` in `PTP.yaml` is empty?

**Comment by xheiop** [comment]
> I started looking how to add a new site, so first not clear thing is why `seedDomSelector` in `PTP.yaml` is empty?

cuz the selector of ptp contains  id of the torrent

**Comment by xheiop** [comment]
> I started looking how to add a new site, so first not clear thing is why `seedDomSelector` in `PTP.yaml` is empty?

If you have any other  questions, you can contact me on telegram at anytime.  My username is `birdplane`
